apps/web/app/config/navigationConfig.ts

```diff
@@ -17,7 +17,7 @@ export interface ParentMenuItem extends BaseMenuItem {
   children: MenuItemType[];
 }
 
-export type MenuItemType = LeafMenuItem | ParentMenuItem;
+export type MenuItemType = LeafMenuItem;
```

Comment:
If we don't need recursive menu items anymore, can you delete the unused types above?